scripts/flash/set-timestamp.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var chalk = require("chalk");
+var displayTime = require("./display-time");
+
+function setTimestamp(augur, numberTicks, address, auth, callback) {
```

Comment:
Why is the new timestamp parameter called `numberTicks`?  Should it be called `newTimestamp` instead?

Reply:
update the parameter